static/app/views/issueDetails/groupEventDetails/groupEventDetails.spec.tsx

```diff
@@ -198,13 +198,19 @@ const mockGroupApis = (
   project: Project,
   group: Group,
   event: Event,
+  replayId: string | undefined,
```

Comment:
nit: Could we make this optional instead of explicitly `undefined`? That way you don't have to add `undefined` to all the callers, only when you want to specify the trace I guess.

```js
replayId?: string;
```

Reply:
yup updated!